lib/galaxy/datatypes/display_applications/parameters.py

```diff
@@ -100,6 +100,8 @@ def _get_dataset_like_object(self, other_values):
         assert data, "Base dataset could not be found in values provided to DisplayApplicationDataParameter"
         if isinstance(data, DisplayDataValueWrapper):
             data = data.value
+        if data.state != data.states.OK:
+            return None
```

Comment:
Are you sure this is necessary ? Both `ready` and `is_preparing` are checking the state in lib/galaxy/datatypes/display_applications/parameters.py. Changing this to None may kick off additional conversions everytime this is being polled on.

Reply:
Yep, that breaks adding a new vcf

Reply:
nevermind, got the wrong x-accel-redirect set up 😓